_posts/2018-01-01-compatibility-layer.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "Compatibility Layer"
+subtitle: "0.1 ❤ 0.3"
+author: "Josef Brandl"
+author_github: "MajorBreakfast"
+date:   2018-01-01
+categories: blog
+---
+
+# Futures 0.1 Compatibility Layer
+
+Rust's futures ecosystem is currently split in two: On the one hand we have the vibrant ecosystem built around [`futures@0.1`][] with its many libraries working on stable Rust and on the other hand there's the unstable [`std::future`][] ecosystem with support for the ergonomic and powerful `async`/`await` language feature. To bridge the gap between these two worlds we have introduced a compatibility layer as part of the [`futures@0.3`][] extension to `std::future`.  This blog post aims to give an overview over how to use it.
+
+[`futures@0.1`]: https://docs.rs/futures
+[`futures@0.3`]: https://rust-lang-nursery.github.io/futures-api-docs/
+[`std::future`]: https://doc.rust-lang.org/nightly/std/future/
+
+## `Cargo.toml`
+
+The compatibility layer can be enabled by setting the `compat` feature in your `Cargo.toml`:
+
+```toml
+futures-preview = { version = "0.3.0-alpha.10", features = ["compat"] }
```

Comment:
`compat` doesn't include `io-compat`, which also pulls in `tokio-io` and adds the `AsyncRead` and `AsyncWrite` compat bits.

Reply:
Yep, I was going to mention the extra features when adding the additional IO/spawn sections.